src/vignette/http/routes.clj

```diff
@@ -137,18 +144,13 @@
         (GET alr/thumbnail-route
              request
              (let [image-params (alr/route->thumb-map (:route-params request))]
-               (if (:unsupported image-params)
-                 (-> (response "unsupported thumbnail route")
-                     (status 307)
-                     (header "Location" (str (env :unsupported-redirect-host "http://images.wikia.com")
-                                             (:uri request))))
-                 (if-let [thumb (u/get-or-generate-thumbnail system image-params)]
```

Comment:
Way back in the day when we didn't support all the legacy url formats, specifically `window-crop` and `window-crop-fixed`. This isn't set anywhere anymore, so no point in keeping it.